IdentityCore/tests/automation/shared/MSIDAutomationTestRequest.h

```diff
@@ -64,7 +64,7 @@
 @property (nonatomic) BOOL ssoExtensionHooksEnabled;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->